swift-service/src/main/java/com/facebook/swift/service/ThriftMethodProcessor.java

```diff
@@ -220,9 +222,11 @@ public void onFailure(Throwable t)
                         } else {
                             contextChain.undeclaredUserException(t);
```

Comment:
But it doesn't cover the case of nifty explicitly sending a TAE. In those places, we don't have access to these contextChain callbacks